### PR TITLE
add new classes CL_4006000 (fibroblast of breast) CL_4006001 (fibroblast skin of scalp)

### PR DESCRIPTION
**Pull request by bonitalam**
This PR addresses issue #422

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -26905,6 +26908,22 @@ EquivalentClasses(obo:CL_3000004 ObjectIntersectionOf(obo:CL_0000111 ObjectSomeV
 SubClassOf(obo:CL_3000004 obo:CL_0000101)
 SubClassOf(obo:CL_3000004 obo:CL_0000111)
 
+# Class: obo:CL_4006000 (fibroblast of breast)
+
+AnnotationAssertion(<http://geneontology.org/formats/oboInOwl#created_by> obo:CL_4006000 <http://orcid.org/0000-0001-9897-3238>)
+AnnotationAssertion(obo:IAO_0000115 obo:CL_4006000 "A fibroblast that is part of the breast.")
+AnnotationAssertion(rdfs:label obo:CL_4006000 "fibroblast of breast")
+EquivalentClasses(obo:CL_4006000 ObjectIntersectionOf(obo:CL_0000057 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000310)))
+SubClassOf(obo:CL_4006000 obo:CL_0000057)
```
Comment (nicolevasilevsky) [inline src/ontology/cl-edit.owl:26917]: add synonym:
breast fibroblast

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -26905,6 +26908,22 @@ EquivalentClasses(obo:CL_3000004 ObjectIntersectionOf(obo:CL_0000111 ObjectSomeV
...
+
+# Class: obo:CL_4006001 (fibroblast of skin of scalp)
+
+AnnotationAssertion(<http://geneontology.org/formats/oboInOwl#created_by> obo:CL_4006001 <http://orcid.org/0000-0001-9897-3238>)
+AnnotationAssertion(obo:IAO_0000115 obo:CL_4006001 "A fibroblast that is part of the skin of scalp.")
+AnnotationAssertion(rdfs:label obo:CL_4006001 "fibroblast of skin of scalp")
+EquivalentClasses(obo:CL_4006001 ObjectIntersectionOf(obo:CL_0000057 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_8300000)))
+SubClassOf(obo:CL_4006001 obo:CL_0000057)
```
Comment (nicolevasilevsky) [inline src/ontology/cl-edit.owl:26925]: add synonym
skin of scalp fibroblast
?

**Review by nicolevasilevsky — APPROVED** [review]
Looks good - suggested minor changes to add synonyms.

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (bonitalam) [reply src/ontology/cl-edit.owl:26925]: Thanks @nicolevasilevsky, I just added synonyms to the terms.

**Review by dosumis — APPROVED** [review]

**Pull request merged**